test/plots/scatterPlotTests.ts

```diff
@@ -275,6 +275,19 @@ describe("Plots", () => {
         svg.remove();
       });
 
+      it("returns undefined from nearestEntity when no scatter points are in view", () => {
```

Comment:
Can we use a consistent name for the tests? `"retrieves undefined from entityNearest when no entities are rendered"` seems best.